### PR TITLE
Map center

### PR DESCRIPTION
**Pull request by katiejiang**
* If no resources, initialize map center to center of US
* Otherwise, map is centered at epicenter of resources
* Issues with current location (lag) so removed that

### CODE REVIEW

**Comment by katiejiang** [comment]
This is ready to merge @HanaPearlman

File: app/assets/scripts/map.js [hunk 1]
```diff
@@ -480,17 +495,18 @@ function populateMarkers(resources) {
...
     oms.addMarker(markers[i]);
     bounds.extend(markers[i].getPosition());
   }
-
+  allResourceBounds = bounds;
   map.fitBounds(bounds);
   map.setCenter(bounds.getCenter());
-  allResourceBounds = bounds;
+  if (markers.length == 1) {
+    map.setZoom(15);
```
Comment (HanaPearlman) [inline app/assets/scripts/map.js]: this is so small but: thoughts on zooming this out a little farther? the super close zoom was a bit disorienting for me.

Comment (katiejiang) [reply app/assets/scripts/map.js]: On it!

Comment (katiejiang) [reply app/assets/scripts/map.js]: Actually this zoom level is good, there's some other weird issue right now, I'll look into it

**Pull request merged**